.github/workflows/push.yml

```diff
@@ -25,6 +25,17 @@ jobs:
         with:
           package-manager: yarn
 
+      - name: Build docs
+        run: |
+          npx typedoc
+          docker run --rm -v ${PWD}:/docs squidfunk/mkdocs-material build
```

Comment:
could we pin this to a version `squidfunk/mkdocs-material:v12345` a specific hash may actually be better.

Reply:
https://hub.docker.com/r/squidfunk/mkdocs-material/tags

Looks like it uses semver, so we could pin it to `9`, `9.4`, `9.4.7`, or the latest hash `91c415cc8084`. Pinning to the major version seems like the best option?

Reply:
Pinned to 9.4 following slack discussion